Source/Core/GeometryPipeline.js

```diff
@@ -2233,13 +2277,31 @@ define([
                 }
 
                 var offsetPoint = Cartesian3.add(intersection, offset, offsetPointScratch);
-                insertSplitPoint(p0Attributes, p0Indices, p0IndexMap, indices, i, p0);
-                insertSplitPoint(p0Attributes, p0Indices, p0IndexMap, indices, -1, offsetPoint);
+                if (defined(applyOffset)) {
```

Comment:
Can you make something similar to `computeTriangleAttributes` or make it more general?